tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -453,6 +480,70 @@ def test_execute(self, mock_hook):
             job_id=job_id, project_id=GCP_PROJECT, location=GCP_LOCATION
         )
 
+    @provide_session
+    def test_operator_extra_links(self, session):
+        job = {}
+        job_id = 'test_job_id_12345'
+        execution_date = datetime(2020, 7, 20)
+        # pylint: disable=line-too-long
+        expected_extra_link = 'https://console.cloud.google.com/dataproc/jobs/{job_id}?region={region}&project={project_id}'.format(     # noqa: E501
+            job_id=job_id,
+            region=GCP_LOCATION,
+            project_id=GCP_PROJECT
+        )
```

Comment:
```suggestion
        expected_extra_link = \
            f'https://console.cloud.google.com/dataproc/jobs/{job_id}?'
            f'region={GCP_LOCATION}&project={GCP_PROJECT}'
```
How about using f-string?

Reply:
Moved the expected link to a variable. Also converted it into f-string. Let me know if there is a better approach.